cmd/alerts/utils/utils.go

```diff
@@ -15,14 +16,35 @@ const (
 	AccountNamespace = "openshift-monitoring"
 	ContainerName    = "alertmanager"
 	LocalHostUrl     = "http://localhost:9093"
-	PodName          = "alertmanager-main-0"
+	PrimaryPod       = "alertmanager-main-0"
+	SecondaryPod     = "alertmanager-main-1"
 )
 
+// ExecInPod is designed to execute a command inside a Kubernetes pod and capture its output.
 func ExecInPod(kubeconfig *rest.Config, clientset *kubernetes.Clientset, cmd []string) (string, error) {
+	var cmdOutput string
+	var err error
 
-	req := clientset.CoreV1().RESTClient().Post().Resource("pods").Name(PodName).
-		Namespace(AccountNamespace).SubResource("exec")
+	// Attempt to execute with the primary pod
+	cmdOutput, err = ExecWithPod(kubeconfig, clientset, PrimaryPod, cmd)
+	if err == nil {
+		return cmdOutput, nil // Successfully executed
+	}
+
+	// If execution with primary pod fails, try with the secondary pod
+	cmdOutput, err = ExecWithPod(kubeconfig, clientset, SecondaryPod, cmd)
+	if err == nil {
+		return cmdOutput, nil // Successfully executed
+	}
 
+	// If execution with both pods fails, print error message
+	fmt.Println("Exec Failed. Please put silence manually")
```

Comment:
I'd remove the println from this util function and leave output to calling functions. Returning the error should be enough for callers to craft their own output to the user.

Reply:
```
fmt.Println("Execution Failed with alertmanager-main-0 and alertmanager-main-1. Please put silence manually")
	return "", er
```
Are we looking something like ^

Reply:
I see now the error is printed out in `ExecWithPod`. I don't love this, as general good practice is to bubble errors up to the caller and let them handle it as they see fit (i.e. functions that perform business logic should only return an error, not print it).

Alas, I don't think that change should be the responsibility of this PR.

Reply:
Negative - I'd remove the Println on line 41 entirely. Let the calling functions handle output to the user.